spec/imsc-hrm.html

```diff
@@ -378,12 +378,11 @@ <h3>How does the HRM measure and limit complexity?</h3>
     <section>
       <h3>Where is the HRM used?</h3>
   
-      <p>The HRM is typically used prior to distribution of the <a>IMSC Document
-      Instance</a> exceeding the HRM limits depend on the context:</p>
-      quality check before distribution.</p>
-  
-      <p>When the HRM is used, the consequences of an <a>IMSC Document
-      Instance</a> exceeding the HRM limits depends circumstances:</p>
+      <p>The HRM is typically used prior to distribution of the <a>IMSC Document Instance</a> to end-user, as an
+      integral part of authoring and as a quality check before distribution.</p>
```

Comment:
Grammar nit, sorry-not-sorry!

```suggestion
      <p>The HRM is typically used prior to distribution of the <a>IMSC Document Instance</a> to the end-user, as an
      integral part of authoring and as a quality check before distribution.</p>
```